### PR TITLE
PIM-4700 Glitches / CSS design dashboard

### PR DESCRIPTION
**Pull request by yulldesign**

### CODE REVIEW

File: src/Pim/Bundle/UIBundle/Resources/public/css/modal.less [hunk 1]
```diff
@@ -4,15 +4,32 @@
 /* Bootstrap modal ****************************************************************************************************/
 
 .modal {
-  .border-radius(8px 8px 6px 6px);
+  border-radius: 0;
+  border: none;
   top: 20%;
+
```
Comment (willy-ahva) [inline src/Pim/Bundle/UIBundle/Resources/public/css/modal.less]: remove blank line

File: src/Pim/Bundle/UIBundle/Resources/public/lib/bootstrap/css/bootstrap.css [hunk 2]
```diff
@@ -5630,16 +5630,15 @@ a.thumbnail:focus {
...
   background-color: #999999;
 }
 
 .label {
-  -webkit-border-radius: 3px;
-     -moz-border-radius: 3px;
-          border-radius: 3px;
+  -webkit-border-radius: 2px;
+     -moz-border-radius: 2px;
+          border-radius: 2px;
```
Comment (willy-ahva) [inline src/Pim/Bundle/UIBundle/Resources/public/lib/bootstrap/css/bootstrap.css]: should be align to the left ?

File: src/Pim/Bundle/UIBundle/Resources/public/css/form.less [hunk 3]
```diff
@@ -216,6 +216,8 @@ a.add-attribute-option {
 .has-switch {
   margin-right: 3px;
   margin-bottom: -6px;
+  border-radius: 0px;
```
Comment (phaseinducer) [inline src/Pim/Bundle/UIBundle/Resources/public/css/form.less]: remove px please

File: src/Pim/Bundle/UIBundle/Resources/public/css/pim.less [hunk 4]
```diff
@@ -149,7 +149,7 @@ a.back-link i {
 
 /** Dashboard ********************************************************************************************************/
 .dashboard {
-  background-color:#f1f1f1;
+  background-color:white;
```
Comment (phaseinducer) [inline src/Pim/Bundle/UIBundle/Resources/public/css/pim.less]: space after colon

File: src/Pim/Bundle/UIBundle/Resources/public/css/pim.less [hunk 5]
```diff
@@ -157,10 +157,8 @@ a.back-link i {
     margin-bottom: 20px;
     &>div.clearfix {
       overflow: hidden;
-      border: 1px solid #b6b6b6;
-      background-color: lighten(@color-light_grey, 4%);
-      .border-radius(5px);
-      .box-shadow(1px 1px 5px rgba(50, 50, 50, 0.2));
+      border: 1px solid #CbCbd2;
```
Comment (phaseinducer) [inline src/Pim/Bundle/UIBundle/Resources/public/css/pim.less]: lowercase

File: src/Pim/Bundle/UIBundle/Resources/public/css/pim.less [hunk 6]
```diff
@@ -174,11 +172,13 @@ a.back-link i {
...
     margin: 0;
+    font-size: 18px;
+    color: @color-new-purple-medium;
+    font-weight: normal;
   }
   &.header {
-    background-color: #fff;
-    padding: 5px 20px;
-    border-bottom: 1px solid #f7f7f7;
+    background-color: #F7F7F9;
```
Comment (phaseinducer) [inline src/Pim/Bundle/UIBundle/Resources/public/css/pim.less]: lowercase

File: src/Pim/Bundle/UIBundle/Resources/public/css/pim.less [hunk 7]
```diff
@@ -188,13 +188,20 @@ hr.small {
...
 /** widgets **********************************************************************************************************/
 .widget-header {
-  padding: 2px 2px 3px 10px;
+  padding: 0 10px;
   font-weight: bold;
-  background: linear-gradient(@backgroundColor-white, @color-light_grey);
   border-bottom: 1px solid #ccc;
+  line-height: 44px;
+  color: @color-new-purple-medium;
+  background-color: #F7F7F9;
```
Comment (phaseinducer) [inline src/Pim/Bundle/UIBundle/Resources/public/css/pim.less]: lowercase

File: src/Pim/Bundle/UIBundle/Resources/public/css/pim.less [hunk 8]
```diff
@@ -212,14 +219,16 @@ a[data-toggle-channel] {
 }
 
 #completeness-widget {
+  width: 100%;
+
   .progress {
     height: 10px;
-    margin-bottom: 5px;
-    background-image: linear-gradient(to bottom, #e4e4e4, #eee);
+    margin-bottom: 0px;
```
Comment (phaseinducer) [inline src/Pim/Bundle/UIBundle/Resources/public/css/pim.less]: no px

File: src/Pim/Bundle/UIBundle/Resources/public/css/pim.less [hunk 9]
```diff
@@ -236,20 +245,26 @@ a[data-toggle-channel] {
     }
   }
   tr.channel {
+    background-color: #ECECF0;
```
Comment (phaseinducer) [inline src/Pim/Bundle/UIBundle/Resources/public/css/pim.less]: lowercase

**Pull request merged**